glusterd2/config.go

```diff
@@ -60,27 +67,8 @@ func parseFlags() {
 // setDefaults sets defaults values for config options not available as a flag,
 // and flags which don't have default values
 func setDefaults() error {
-	cwd, err := os.Getwd()
```

Comment:
This is still required to set `workdir` and `localstatedir` properly below.

Reply:
why? both workdir and localstatedir are set using LDFLAGS

Reply:
If workdir is not explicitly set by user, it (rightly) defaults to current working directory which is only available during run time via `Getwd()`

Reply:
I am missing something. both `workdir` and `localstatedir` are defaulted to `$prefix/var/lib/glusterd2` now. What is the expected default path for workdir?

Reply:
Ok, is it when user configures `localstatedir` but not `workdir`? if so is it ok to set `workdir=localstatedir` when not specified

Reply:
> What is the expected default path for workdir?

Current working directory.